text/0649-deprecation-staging.md

```diff
@@ -155,10 +155,13 @@ Stepping through the new options individually:
   assertions, depending on the app or addon configuration. This is explained
   more below.
 
-If the `since` key is not provided (as is the case for existing deprecations),
-the deprecation is assumed to be `enabled`. This RFC also... wait for it... deprecates
-the usage of the `deprecate()` function without a `since` key. This meta deprecation
-will be enabled at the same time as it becomes available like this:
+  If the `since` key is not provided (as is the case for existing
+  deprecations), the deprecation is assumed to be `enabled`.
+
+This RFC also... wait for it... deprecates the usage of the `deprecate()`
+function without the `for` or `since` keys. This meta deprecation will be
```

Comment:
Shouldn't the example below also include a `for` key? (TBH, it's a little confusing as to what this example is attempting to demonstrate.)

Reply:
Yeah, this is what was there before and I was trying to make only minimal changes.

Reply:
The snippet below shows the implementation of a _missing_ `since` deprecation, using the syntax that is introduced in this RFC. It's... confusing to say the least 😢